### DISCUSSION TITLE
コントリビューター40人以上を超えた場合の表示方法

### DISCUSSION THREAD

**Discussion by pss-aileen** [first-contributions-ja/first-contributions-ja.github.io#280] [12: 新機能の提案]
#279

こちらでIssueに投稿していただいたのですが、
コントリビューターが大変ありがたいことに40人を超えたので
ゆるりゆるりと今後どうするかを決めておいた方が良いのかなと思いました。

**Comment by pss-aileen** [comment]
個人的にどうしたいか

- 気持ちは全員分表示したい
- でも、多くなればなるほど表示速度が落ちそう、ただまだ今は大丈夫..？
- なので何か別で全員を表示して、50人とか、制限を設けて、それを超えたら表示させないようにする...など

**Reply by kazzyfrog** [reply]
自分も、全員分表示できたらいいなと思ってます！

ジャストアイデアですが、
フッターを、少し作り直してもいいかもですね！

例えば、そこに、smallサイズの絵文字を、30個表示できるようにするとか！

なんか下に溜まっている感じで、いいかなと！

**Reply by pss-aileen** [reply]
> なんか下に溜まっている感じで、いいかなと！

これいいですね！賛成です！👍

**Reply by kazzyfrog** [reply]
そしたら、時間がある時に、フッターデザイン考えてみますね！
- 表示する個数
- フッターでもアニメーションする？場合は、既存のセクションと同じか変えるかなど！

色々試してみます！👍

**Reply by pss-aileen** [reply]
> そしたら、時間がある時に、フッターデザイン考えてみますね！

すみません、ありがとうございます！！

**Comment by kazzyfrog** [comment]
## 現状、その他の問題・気になること：
思いつき次第、追加していきます！他にあれば、追加してくださいー！

- 好きな色が白（#fff）、もしくはそれに近い色の場合に、safariで絵文字と背景デザインが見えなくなってしまう問題。

**Reply by pss-aileen** [reply]
最初の方にContributors.jsonに追加してくださった方で、IDを変えたのかプロフィール画像が表示されない方がいるので、そういった場合何か他の画像に差し替えられないか...と思っています。

![スクリーンショット 2024-08-20 11 09 29](https://github.com/user-attachments/assets/7425f42a-6c6c-4194-ab2d-0e75cd4108d7)
